### DISCUSSION TITLE
How do I increase the distance that towns activate at?

### DISCUSSION THREAD

**Discussion by UnRealxInferno** [Vdauphin/HeartsAndMinds#1380] [Question and answer] [answered]
How do I increase the distance that towns activate at?

The parameters aren't very clearly labelled 😅

**Comment by Vdauphin** [comment, marked as answer]
Hello,
https://github.com/Vdauphin/HeartsAndMinds/blob/master/%3DBTC%3Dco%4030_Hearts_and_Minds.Altis/core/def/param.hpp#L410-L415
Cheers
